recipes/license_finder.rb

```diff
@@ -19,3 +19,15 @@
       - 'package.json'
   HEREDOC
 end
+
+after_bundle do
+  bundle_command 'exec license_finder permitted_licenses add "New BSD" "Simplified BSD" ruby "2-clause BSDL" ' \
+                 '"Python-2.0" MIT "Apache 2.0" ISC CC0-1.0 CC-BY-3.0 "Mozilla Public License 2.0" LGPL MIT-0 ' \
+                 '"Brakeman Public Use License" Apache "(CC-BY-4.0 AND OFL-1.1 AND MIT)" CC-BY-4.0 ' \
+                 '"MPL-2.0 OR Apache-2.0" "(MIT AND Zlib)" "(WTFPL OR MIT)" "PayPal SDK License" ' \
+                 '"MIT (http://mootools.net/license.txt)" "(MIT AND CC-BY-3.0)" 0BSD BSD '
```

Comment:
I'd extract the licenses to an array for easier managing:
```ruby
bundle_command "exec license_finder permitted_licenses add #{permitted_licenses.join(' ')}"

Reply:
Sounds reasonable, let's do it in a separate PR